src/distributors/local-installer.ts

```diff
@@ -0,0 +1,60 @@
+import * as tc from '@actions/tool-cache';
+
+import fs from 'fs';
+import path from 'path';
+import semver from 'semver';
+import { macOSJavaContentDir, setupFromJdkFile } from '../util';
+
+import { JavaBase } from './base-installer';
+import {
+  JavaInstallerOptions,
+  JavaDownloadRelease,
+  JavaInstallerResults
+} from './base-models';
+
+export class LocalDistributor extends JavaBase {
+  private jdkFile: string;
+  constructor(installerOptions: JavaInstallerOptions) {
+    super('local_distributor', installerOptions);
```

Comment:
`local_distributor` -> `LocalJDKFile`

So the folder name will be `Java_LocalJDKFile_jdk`

Reply:
Done. Thank you.